tests/monotouch-test/CoreBluetooth/UuidTest.cs

```diff
@@ -35,7 +39,11 @@ public void Roundtrip_16bits ()
 			using (CBUUID uuid = CBUUID.FromString ("1234")) {
 				Assert.That (uuid.Handle, Is.Not.EqualTo (IntPtr.Zero), "Handle");
 				Assert.IsNotNull (uuid.Data, "Data");
-				var expected = UIDevice.CurrentDevice.CheckSystemVersion (7,1) ? "1234" : "Unknown (<1234>)";
+#if MONOMAC
+				var expected = "Unknown (<1234>)";
+#else
+				var expected = UIDevice.CurrentDevice.CheckSystemVersion (7, 1) ? "1234" : "Unknown (<1234>)";
```

Comment:
This one is returning the "Unknown (<1234>)" on macOS 10.12.  Changing the CheckSystemVersion to CheckXcodeVersion would have it looking for "1234"